tests/unit/stream_alert_alert_processor/test_outputs/test_pagerduty.py

```diff
@@ -251,6 +250,8 @@ def test_policy_verify_success_default(self, get_mock):
         assert_equal(policy_verified['id'], 'good_policy_id')
         assert_equal(policy_verified['type'], 'escalation_policy_reference')
 
+    @patch('stream_alert.alert_processor.outputs.output_base.OutputDispatcher.MAX_RETRY_ATTEMPTS',
```

Comment:
Can you instead patch this on the entire `TestPagerDutyIncidentOutput` instead of individual methods? :)